### DISCUSSION TITLE
Come one, uBo team, confirm that you are hired in Mozilla!

### DISCUSSION THREAD

**Discussion by tedomedo** [uBlockOrigin/uBlock-issues#3434] [General] [answered]
Whoever wants uBO on android, you Mozilla workers always reply in the same way: "If you want uBo on android, install Firefox." You're not allowed to create uBo.apk, right? Don't reply with text: "We are free, we don't work in Mozilla. " Reply with action: create .apk and place it on google play store.

**Comment by garry-ut99** [comment]
Your conspiracy theory doesn't seem to be true, the reason why they're telling it, is because uBO is a browser extension, not a standalone app, it never was a standalone app, and is not going to be a standalone app.
- https://github.com/uBlockOrigin/uBlock-issues/discussions/3023
- https://github.com/uBlockOrigin/uBlock-issues/discussions/3416

If you mean that it never was a standalone app because uBlock is hired in Mozilla since the beginning, then here is another proof that uBlock is not hired in Mozilla, the spat between uBlock and Mozilla:
- https://github.com/uBlockOrigin/uBOL-home/issues/197
- https://github.com/uBlockOrigin/uBOL-home/discussions/231

**Reply by tedomedo** [reply]
But uBo team looks so very limited or controlled by someone, that's why the team doesn't want to/isn't allowed to create app. This team looks like AI. Let us know why uBo team doesn't want to or can't create app. Is Mozilla giving cash to uBo team while uBo is only addon?
Your reply is the same: it was and will be only add-on. As a dictator is saying: "There was and will be only my rules, obey them or f... off!"

**Reply by stephenhawk8054** [reply]
> why the team doesn't want to/isn't allowed to create app

Because we don't have time and motivations? If you have time and motivations, feel free to do it. uBO is a GPLv3 project, feel free to use the code and develop any GPLv3 projects yourself.

Why are we demanded to give reasons to every random person for something we don't want to do? Are we your employees? Do we get paid for sitting here, working, coding and wasting our time to satisfy someone else's demand?

**Reply by garry-ut99** [reply]
> Your reply is the same: it was and will be only add-on. As a dictator is saying: "There was and will be only my rules, obey them or f... off!"

My reply was not my own reply, nor was it my  decision, nor my opinion, I'm a volunteer, I only repeated / quoted facts: uBO team's decisions & answers from the previous threads, that's why it looks same or similiar.

**Reply by tedomedo** [reply]
> 
> Because we don't have time and motivations? If you have time and motivations, feel free to do it. uBO is a GPLv3 project, feel free to use the code and develop any GPLv3 projects yourself.
> 

You have time to add new filters few times but don't have time to create .apk? Whatta joke!

**Reply by uBlock-user** [reply]
@tedomedo You have time to whine, bitch and moan about .apk, so create one yourself and maintain it first.

**Comment by uBlock-user** [comment, marked as answer]
Because gorhill doesn't want to. How dumb do you have to be to not realise that  by now ?

**Reply by tedomedo** [reply]
> Because gorhill doesn't want to. How dumb do you have to be to not realise that by now ?

I know that long time and uBo dorks are constantly proving that.

**Reply by uBlock-user** [reply]
> I know that long time

So why bother opening this dumb topic and why come back to prove yourself dumb again ?

**Comment by Yuki2718** [comment]
If converting a JS extension to a java-based mobile app is such an easy task, why don't you do by yourself? uBO is open source so nothing stops that for you.
